config/config.go

```diff
@@ -125,6 +125,19 @@ type ExchangeConfig struct {
 	PairsLastUpdated          int64                     `json:",omitempty"`
 	ConfigCurrencyPairFormat  *CurrencyPairFormatConfig `json:"ConfigCurrencyPairFormat"`
 	RequestCurrencyPairFormat *CurrencyPairFormatConfig `json:"RequestCurrencyPairFormat"`
+	InternationalBank         BankConfig
+	LocalBank                 BankConfig
+	DepositBank               BankConfig
+}
+
+// BankConfig is a single bank configuration
+type BankConfig struct {
+	Enabled       bool
+	BankName      string
+	AccountName   string
+	AccountNumber string
+	BSBNumber     string
+	SwiftCode     string
```

Comment:
An IBAN number would be great to include as it is just one number used for cross boarder payments which includes the country, swift and account number.